credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -4,6 +4,7 @@
 import string_utils
 
 from ..base_model import BaseModel
+from difflib import SequenceMatcher
```

Comment:
this is part of the standard library, so it shouldn't be placed here [ref](https://google.github.io/styleguide/pyguide.html#313-imports-formatting)